packages/service-library/src/servicelib/container_utils.py

```diff
@@ -77,10 +93,10 @@ async def run_command_in_container(
             _execute_command(container_name, command), timeout
         )
     except DockerError as e:
-        if e.status == status.HTTP_404_NOT_FOUND:
+        if e.status == _HTTP_404_NOT_FOUND:
```

Comment:
we have a status that you can import in servicelib I believe. please use this instead of creating even more aliases. thanks

Reply:
if you use fastapi, then please use the status from fastapi